pages.es/common/chown.md

```diff
@@ -1,24 +1,28 @@
 # chown
 
-> Cambia la propiedad de usuario y grupo sobre archivos y directorios.
+> Cambia la propiedad de usuario y grupo de archivos y directorios.
```

Comment:
Cambia el usuario y grupo propietarios de archivos y directorios.

To make it clearer ownership.